### PR TITLE
Language.html.jinja2: Remove hyperlink if None

### PR DESCRIPTION
**Pull request by rajdeepbharati**
Fixes https://github.com/coala/coAST/issues/70

### CODE REVIEW

File: templates/Language.html.jinja2 [hunk 1]
```diff
@@ -14,9 +14,13 @@
     <tr>
       <th>Grammar File</th>
       <td>
+        {% if language.grammar_file %}
         <a href = {{ language.grammar_file }}>
           {{ m.show_grammar(language.grammar_file) }}
         </a>  
+        {% else %}
+          {{ m.show_grammar(language.grammar_file) }}
+        {% endif %}
```
Comment (jayvdb) [inline templates/Language.html.jinja2]: Unlabeled control end tag

*Origin: Jinja2Bear, Section: `jinja2`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpo67x_gqp/templates/Language.html.jinja2
+++ b/tmp/tmpo67x_gqp/templates/Language.html.jinja2
@@ -20,7 +20,7 @@
         </a>  
         {% else %}
           {{ m.show_grammar(language.grammar_file) }}
-        {% endif %}
+        {% endif %}{# if language.grammar_file #}
       </td>
     </tr>    
     <tr>
```

Comment (rajdeepbharati) [reply templates/Language.html.jinja2]: Done

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/75/commits/4cb511e3545449a33a80c139176790558006076f

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**